### PR TITLE
Use Synapse Workshop to edit Getting Started Guide 

### LINKED ISSUE #579: Shorten Getting Started Guide
[Guide](https://docs.synapse.org/articles/getting_started.html) could be condensed, and thus more user friendly, even still. 

@ankimno what was confusing and distracting? Was there information you were hoping to find that is missing from this document?

### PR DESCRIPTION
**Pull request by kelshmo**
Closes #579. Here is a major revision of the Getting Started Guide to make language more accessible, concise and conversational. Most of the language is from the Synapse Workshop powerpoint.

I would like to discuss the benefit/utility of adding images to this page given the added cost of maintaining this type of content. For example, to add a "Navigating Tools Menu" section requires images.

### CODE REVIEW

File: articles/getting_started.md [hunk 1]
```diff
@@ -6,99 +6,90 @@ category: intro
 order: 1
 ---
 
-# Get Started with Synapse
+# Getting Started with Synapse
```
Comment (kdaily) [inline articles/getting_started.md]:
```suggestion
```

Now that we have the ability to get titles from YAML header.

File: articles/getting_started.md [hunk 2]
```diff
@@ -6,99 +6,90 @@ category: intro
...
+## Synapse IDs
 
-## Organizing Data: creating Files and Folders
+Synapse Projects are assigned a Synapse ID, a globally unique identifier used for reference with the format `syn12345678`. Often abbreviated to "synID", the ID of an object never changes, even if the name does. The Synapse ID is always accessible in the URL and visible on the webpage. 
 
-You can use the `Files` tab to upload data, code, results, and other information to your project. Synapse Folders are used to organize content and can contain other Folders and Files. Synapse `Folders` and `Files` are also assigned a unique identifier (Synapse ID).
+## Organizing Content in `Files` and `Folders`
 
-To add a `Folder`:
+Projects contain Files, which can be organized into Folders. Folders and Files also have their own unique Synapse IDs and can be moved within or across Projects. Uploaded files are stored within Synapse storage.
```
Comment (kdaily) [inline articles/getting_started.md]:
```suggestion
Projects contain Files, which can be organized into Folders. Folders and Files also have their own unique Synapse IDs and can be moved within or between Projects. Uploaded files are stored within Synapse storage.
```

**Review by kdaily — CHANGES_REQUESTED** [review]
minor changes, but looks good to me.

File: articles/getting_started.md (hunk 1 above)
Comment (kdaily) [reply articles/getting_started.md]: Fixed via merge with `gh-pages`.

**Comment by kelshmo** [comment]
@ryanluce72 I'm going to merge this article since I need these changes to be live by this Friday, but it would be great for you to review the language and feel of our "Getting Started Guide" to ensure continuity across all our user-facing material!

**Pull request merged**